docs/versions.rst

```diff
@@ -55,3 +55,13 @@ for example ``http://pip.readthedocs.io/``,
 they will be redirected to the **Default version**.
 This defaults to **latest**,
 but could also point to your latest released version.
+
+Version warning
+---------------
+
+This is the banner with the text *You are not using..*
+that is displayed on older versions of your docs (not stable or latest),
+redirecting the users to the latest version of your docs.
```

Comment:
Now, thinking a little more on this, I think we need to be more clear here on what is this exactly since it's disabled by default. Consider that the user maybe haven't seens this message before and we need to communicate what it this banner, how it will looks like and when it will appear.

Considering those points, could you expand these docs a little more?